pkg/virt-handler/vm.go

```diff
@@ -2934,3 +2950,37 @@ func nodeHasHostModelLabel(node *k8sv1.Node) bool {
 	}
 	return false
 }
+
+func (d *VirtualMachineController) reportDedicatedCPUSetForMigratingVMI(vmi *v1.VirtualMachineInstance) error {
+	baseCPUSetPath := "/proc/1/root/sys/fs/cgroup/cpuset"
```

Comment:
I am afraid this will not wok on the systems with cgroups v2 since the path will be different (smth like `/sys/fs/cgroup/<slice>/cpuset.cpus`). Maybe it would be better to use here the helpers from `pkg/virt-handler/cgroup`, e.g.:

https://github.com/kubevirt/kubevirt/blob/d10bcd59ff1a819dc66457b8a6775e702362ccca/pkg/virt-handler/cgroup/cgroup.go#L50-L52

I think it can be easiliy adapted to support also the slice argment and the implementation here will support cgroups v1 and v2.

Reply:
Good point, thank you!
I wrote and pushed a fix, unfortunately there's only 1 lane that can run my functional test and it appears to use cgroups v1...

Reply:
Please note this on-going PR: https://github.com/kubevirt/kubevirt/pull/6042.
This PR performs major changes for cgroups, introducing a completely new cgroups package that is capable of dealing with v1/v2 agnostically.

Therefore I wouldn't work too hard to support v2 on your part ATM. After this PR kicks in we can use a method that will give you the cpu set without dealing with files and paths.